server/services/nl.py

```diff
@@ -16,7 +16,7 @@
 from collections import OrderedDict
 import logging
 import re
-from typing import Dict, List, Union
+from typing import Dict, List, Tuple, Union
```

Comment:
nit: is Tuple still used (in other file too)

Reply:
done